sia/src/types/v1.rs

```diff
@@ -883,25 +912,111 @@ mod tests {
     #[test]
     fn test_serialize_transaction() {
         let transaction = Transaction {
-            siacoin_inputs: Vec::new(),
-            siacoin_outputs: Vec::new(),
-            file_contracts: Vec::new(),
-            file_contract_revisions: Vec::new(),
-            storage_proofs: Vec::new(),
+            siacoin_inputs: vec![SiacoinInput {
+                parent_id: SiacoinOutputID::parse_string(
+                    "750d22eff727689d1d8d1c83e513a30bb68ee7f9125a4dafc882459e34c2069d",
+                )
+                .unwrap(),
+                unlock_conditions: UnlockConditions {
+                    timelock: 0,
+                    public_keys: vec![UnlockKey::parse_string(
+                        "ed25519:800ed6c2760e3e4ba1ff00128585c8cf8fed2e3dc1e3da1eb92d49f405bd6360",
+                    )
+                    .unwrap()],
+                    signatures_required: 6312611591377486220,
+                },
+            }],
+            siacoin_outputs: vec![SiacoinOutput {
+                value: Currency::new(890415399000000000000000000000000),
+                address: Address::parse_string(
```

Comment:
I'd search for all the spots where we still do this and use the cool new macro instead.